.github/workflows/sanitizer.yml

```
@@ -0,0 +1,45 @@
+name: Sanitize
+
+on:
+  workflow_dispatch:
+
+jobs:
+  cargo-careful:
+    runs-on: ubuntu-latest
+    container:
+      image: osgeo/gdal:ubuntu-full-3.6.0
```

```suggestion
      image: ghcr.io/osgeo/gdal:ubuntu-full-3.6.4
```

GDAL is no longer updated on Docker Hub since their free team account shenanigans.